client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -62,61 +69,41 @@ class DeployedModel(object):
 
     """
 
```

Comment:
Do you have any thoughts on having a

```suggestion
_PREDICTION_ID_HEADER_NAME = "verta-request-id"

```

? I've had people recommend to me using constants for commonly-used strings to help with clarity and avoiding typos, but I personally go either way.